### DISCUSSION TITLE
Does this remote control solution not support Debian systems in pure command-line mode well?

### DISCUSSION THREAD

**Discussion by yzazzx2021** [rustdesk/rustdesk#6396] [Q&A]
I plan to unify the remote control scheme for all devices on my end,Replace all with Rustdesk and have completed the Windows graphical deployment control,Today, I plan to directly manage the remaining two Proxmox virtualization platforms, which are based on a Debian distribution system. Therefore, I used the installation package of Deb. After installing the client software, I found that neither the program directory nor the service files were registered with my own server, nor could I change the fixed password in any of them, So is it a Linux server system that has not yet been adapted to pure command-line mode?

**Comment by basilgello** [comment]
Look at Debian Dockerfile [here](https://github.com/rustdesk/rustdesk/issues/6360#issuecomment-1807080455)

**Comment by yzazzx2021** [comment]
Thank you for your reminder, but I'm afraid you haven't figured out a problem. If you force the Docker container to be installed on a fully loaded Proxmox virtualization platform just for a single goal, it may waste the remaining computing resources, which is very bad.

**Reply by basilgello** [reply]
No, I meant you can check this minimal example on how to run Rustdesk on Xvfb in your Debian OS :)

**Comment by yzazzx2021** [comment]
I have an immature proposal, which means that you can refer to the many one click installation scripts on GitHub and create a response script that belongs to the pure command line mode of Rustdesk,It includes two aspects of modification: the first is the password aspect, which includes whether to choose to view dynamic passwords, whether to abolish dynamic passwords and change them to fixed passwords, and the second is to register with a custom server.

**Reply by basilgello** [reply]
Makes sense! But there is already `--password` and `--headless` switch so best to add one more to control dynamic passwords.
